packages/zoe/src/contractFacet/offerHandlerStorage.js

```diff
@@ -0,0 +1,27 @@
+import { makeWeakStore as makeNonVOWeakStore } from '@agoric/store';
+
+import { makeHandle } from '../makeHandle';
+
+export const makeOfferHandlerStorage = () => {
+  /** @type {WeakStore<InvitationHandle, (seat: ZCFSeat) => unknown>} */
```

Comment:
Since this is internal to this module, and the module only uses `OfferHandler` opaquely, I'm curious why you chose not to say 
```suggestion
  /** @type {WeakStore<InvitationHandle, OfferHandler>} */
```

I do see the stricter `unknown` rather than `OfferHandler`'s `any`. But I can't imagine how it could make a difference in this module. In any case, there's no `// @ts-check` in this file. But adding it doesn't seem to make a difference anyway.

Rather than effectively loosen the return type from `unknown` to `any`, at https://github.com/Agoric/agoric-sdk/pull/3158 I experiment with going the other way, changing `OfferHandler` to return `unknown`. We'll see how it does under CI. In any case, I'm not suggesting that typing change in this PR, though feel free to do it if you like it. I'm suggesting only the typing changes to this file.

Reply:
It turns out that we must use `any`, since `unknown` means there is a return value, which is *not* a requirement of the offerHandler. With `unknown`, I get the error:

```
A function whose declared type is neither 'void' nor 'any' must return a value.
```

Reply:
`any` is fine.